Jenkinsfile.groovy

```diff
@@ -91,7 +91,10 @@ spec:
 
         stage('Deploy to argo-dev') {
             when {
-                branch "develop"
+                anyOf {
+                    branch "develop"
+                    branch "test-programs"
```

Comment:
```suggestion
```

to be removed as soon as the PR is approved. I left it in case there's feedback.